test/unit-test-helpers/AnalyticsFrameworkTestUtils.js

```diff
@@ -211,9 +211,14 @@ if (!OO.Analytics.Utils)
           autoPlay = innerMetadata.autoPlay;
         }
 
-        plugin.processEvent(OO.Analytics.EVENTS.VIDEO_PLAYER_CREATED, [{
-          pcode: metadata.pcode,
-          playerBrandingId: metadata.playerBrandingId
+        plugin.processEvent(OO.Analytics.EVENTS.VIDEO_PLAYER_CREATED, [{ 
+          params:{
+            pcode: metadata.pcode,
+            playerBrandingId: metadata.playerBrandingId
+          },
+          embedCode: metadata.embedCode,
+          playerUrl: metadata.playerUrl,
+          playerCoreVersion : "v4"
```

Comment:
Do we need to update the VIDEO_PLAYER_CREATED documentation with these new params?

Reply:
These aren't actually extra parameters, they are properties defined in the object. The only parameter will still be a single array of objects, as with the other events. Most of those events are similarly defined, with "paramArray Array of length 1, containing the original parameters" being the only parameter described in the docs. None any member attributes of the objects within the array are mentioned